package.nix

```diff
@@ -4,6 +4,7 @@
   makeWrapper,
   e2fsprogs-nofortify,
   erofs-utils,
+  fetchFromGitHub,
```

Comment:
I do not understand this commit b91550e8e1e8d230915740af945b0cb12d8bd7a4 - the message and the content are not in sync.

Maybe the main change was already committed earlier, and only this line change left?

Reply:
rebase gone wrong, will fix once @rxpha3l takes care of the other points